### ISSUE TITLE
blackjax seems to have arguments changed

### ISSUE THREAD

**Issue by ferrine** [pymc-devs/pymc#6921]
### Description

```
Changing jax-0.4.13-pyhd8ed1ab_0 ==> jax-0.4.16-pyhd8ed1ab_0
Linking chex-0.1.83-pyhd8ed1ab_0
Linking jaxopt-0.8-pyhd8ed1ab_0
Linking optax-0.1.7-pyhd8ed1ab_0
Linking blackjax-1.0.0-pyhd8ed1ab_0
```
```python
adapt = blackjax.window_adaptation(
            algorithm=algorithm,
            logprob_fn=logprob_fn,
            num_steps=tune,
            target_acceptance_rate=target_accept,
        )
E       TypeError: window_adaptation() missing 1 required positional argument: 'logdensity_fn'
E       --------------------
E       For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
```

**Comment by junpenglao** [comment]
Yep: https://blackjax-devs.github.io/blackjax/autoapi/blackjax/adaptation/window_adaptation/index.html#blackjax.adaptation.window_adaptation.window_adaptation

```python
def window_adaptation(
    algorithm: Union[mcmc.hmc.hmc, mcmc.nuts.nuts],
    logdensity_fn: Callable,
    is_mass_matrix_diagonal: bool = True,
    initial_step_size: float = 1.0,
    target_acceptance_rate: float = 0.80,
    progress_bar: bool = False,
    **extra_parameters,
)
```

**Comment by ferrine** [comment]
how to access in the inference results now?
```
AttributeError: 'HMCState' object has no attribute 'potential_energy'
```

**Comment by ferrine** [comment]
I guess it was renamed to logdensity

**Comment by ferrine** [comment]
closed by #6922
